libraries/helpers.rb

```diff
@@ -81,7 +81,7 @@ def merge_confluence_settings
       when 'postgresql'
         settings['database']['port'] ||= 5432
       else
-        raise "Unsupported database type: #{settings['database']['type']}"
+        warn "Unsupported database type: #{settings['database']['type']} - Use a supported type or handle DB creation/config in a wrapper cookbook!"
```

Comment:
Frankly speaking, this PR adds `hsqldb` as a supported database type, so such warning message doesn't look correct. I'd keep `raise` here as is and just add `hsqldb` as a no-op value like you did in `database` recipe.